src/utils/init/config.js

```diff
@@ -0,0 +1,37 @@
+const chalk = require('chalk')
+
+const configGithub = require('./config-github')
+const configManual = require('./config-manual')
+
+const logSuccess = ({ log, repoData }) => {
+  log()
+  log(chalk.greenBright.bold.underline(`Success! Netlify CI/CD Configured!`))
+  log()
+  log(`This site is now configured to automatically deploy from ${repoData.provider} branches & pull requests`)
+  log()
+  log(`Next steps:
+
+  ${chalk.cyanBright.bold('git push')}       Push to your git repository to trigger new site builds
+  ${chalk.cyanBright.bold('netlify open')}   Open the Netlify admin URL of your site
+  `)
+}
+
+const configureRepo = async ({ context, siteId, repoData, manual }) => {
+  if (manual) {
+    await configManual({ context, siteId, repoData })
+  } else {
+    switch (repoData.provider) {
```

Comment:
Would it be clearer to use `else if`?

Reply:
Done in https://github.com/netlify/cli/pull/1619/commits/31b98294866a87801980d981cfff7dd6f1d18c8f